main/main.cpp

```diff
@@ -362,9 +362,9 @@ void Main::print_help(const char *p_binary) {
 #ifdef DEBUG_METHODS_ENABLED
 	OS::get_singleton()->print("  --gdnative-generate-json-api     Generate JSON dump of the Godot API for GDNative bindings.\n");
 #endif
+#endif
 #ifdef TESTS_ENABLED
 	OS::get_singleton()->print("  --test [--help]                  Run unit tests. Use --test --help for more information.\n");
-#endif
 	OS::get_singleton()->print("\n");
```

Comment:
Why? This won't print a newline with `tests=no`.

Reply:
Looking at the other categories makes me realize that that last return should _always_ be enabled for formatting and consistency purposes, and yet it is not. I moved it out of the ifdef blocks as a result.